### PR TITLE
Tutorial for EBL Markers

### PR DESCRIPTION
**Pull request by ThomasGrottke**
Short tutorial for EBL markers added.

### CODE REVIEW

File: docs/tutorial/tutorial.rst [hunk 1]
```diff
@@ -546,6 +547,85 @@ Also note, that :func:`.GridLayout.generate_layout` returns `two` values. We hav
 ``layout_cell``. The value in ``mapping`` will tell you where each device was placed. To make use of this, you have to
 pass a unique id when calling ``add_to_row``.
 
+Generating electron beam lithography markers
+============================================
+
+When writing several layers with electron beam lithography, markers are needed to align these layers. There is a class
+in gdshelpers that will help you to generate these markers. Note that at the moment only square markers can be found
```
Comment (HelgeGehring) [inline docs/tutorial/tutorial.rst]: Could you mention here, that for other types of markers one would need to write an own `frame_generator` and can then use the same class? Thanks!

File: docs/tutorial/tutorial.rst [hunk 2]
```diff
@@ -546,6 +547,85 @@ Also note, that :func:`.GridLayout.generate_layout` returns `two` values. We hav
 ``layout_cell``. The value in ``mapping`` will tell you where each device was placed. To make use of this, you have to
 pass a unique id when calling ``add_to_row``.
 
+Generating electron beam lithography markers
+============================================
```
Comment (HelgeGehring) [inline docs/tutorial/tutorial.rst]: Maybe start with explaining how markers can individually be placed using `add_ebl_marker`? The `evl_frame` in just uses this function for a given layout

Comment (ThomasGrottke) [reply docs/tutorial/tutorial.rst]: done

File: docs/tutorial/tutorial.rst (hunk 1 above)
Comment (ThomasGrottke) [reply docs/tutorial/tutorial.rst]: done

File: docs/tutorial/tutorial.rst (hunk 2 above)
Comment (ThomasGrottke) [reply docs/tutorial/tutorial.rst]: done

**Comment by HelgeGehring** [comment]
Looks good, thanks!

**Pull request merged**